zombies.js

```diff
@@ -79,6 +90,99 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this._pack = [];
+  this._maxHealth = this.health;
```

Comment:
`_pack` and `_maxHealth` should be on const instead of this for truly private variables.